src/Microsoft.Health.Dicom.Api/Features/Routing/KnownRoutes.cs

```diff
@@ -12,6 +12,7 @@ public static class KnownRoutes
         private const string InstancesRouteSegment = "instances";
         private const string MetadataSegment = "metadata";
         private const string ExtendedQueryTagsRouteSegment = "extendedquerytags";
+        private const string VersionSegment = "v{version:apiVersion}";
```

Comment:
Wondering if there are any thoughts on if we want it to be 
service_url/api/v1
or
service_url/v1

Reply:
I hadn't thought about that. I'm ok with either way, although the non-versioned url would not have the /api in it since it needs to stay unchanged

Reply:
Yea I've seen APIs with and without the /api/. 
@smithago is there any preference for either.

Reply:
Personally, since we have the OHIF viewer optionally at the root, it may make sense to keep `/api` in the path to direct users to our REST APIs.

Reply:
Makes sense to me! I've added /api to the versioned route

Reply:
Actually I have removed /api. We discussed at standup, and decided that since PAAS is only APIs and the OHIF viewer is just for testing purposes, it doesn't add value to have a distinct API route